addons/mail/static/tests/tours/discuss_channel_call_public_tour.js

```diff
@@ -22,7 +22,8 @@ registry.category("web_tour.tours").add("discuss_channel_call_public_tour.js", {
         {
             content: "Check that the call has started",
             trigger: ".o-discuss-Call",
-            run() {
+            async run() {
+                await new Promise((r) => setTimeout(r, 250));
```

Comment:
What makes 250 enough?

Reply:
IMO I'd prefer a DOM state mutation (UI visible or not... former preferred ofc) or a selfSessionProm awaiting.

The `await 250` is not the best, but we can relatively deduce from the run below that `o-discuss-Call` and `selfSession` requires some time to match. The DOM state mutation awaiting feels artificial, but would be more robust to race conditions.

Reply:
Instead of checking the state of the rtc_service, I replaced with a check on the `Disconnect` button from the call context menu, it should only be there if `rtc_service.state.channel` is the current channel, which is equivalent to checking the `selfSession` as channel is set at the same time (when the RPC returns)